packages/components/src/internal/dropdowns.tsx

```diff
@@ -83,28 +84,32 @@ interface DropdownAnchorProps extends PropsWithChildren {
  * See docs in docs/dropdowns.md
  */
 export const DropdownAnchor: FC<DropdownAnchorProps> = props => {
-    const { children, label, pullRight, title } = props;
+    const { children, label, pullRight, title, asAnchor = true } = props;
```

Comment:
Same naming comment here as for the `AddEntitiesButton`: you can create an anchor that's not an anchor, which is not intuitive. `DropdownActionItem` is probably too generic, though.

Reply:
I renamed this to `DropdownMenu` and left in the `asAnchor` prop